### PR TITLE
Update pingdom. events only in integration status monitor

### PR DESCRIPTION
**Pull request by technovangelist**

### CODE REVIEW

File: content/integrations/pingdom.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ At this time we track the response_time metric for any sites you configure on th
     (If you have a Team account, you can use your own credentials and specify the account you wish to pull checks from.)
 1.  You can ignore some checks by unchecking them or add some tags to the events that are going to be generated.
 
-Note: Pingdom does not provide an API for transaction checks, so we’re not able to show them in Datadog. 
+Note: Pingdom does not provide an API for transaction checks, so we’re not able to show them in Datadog.
```
Comment (CharlyF) [inline content/integrations/pingdom.md]: I think this is now available @technovangelist:  https://www.pingdom.com/resources/api#ResourceProbes

Comment (kmshultz) [inline content/integrations/pingdom.md]: Also, nit, while "we" is okay for blog posts, we shouldn't use it in technical Docs (in my opinion). 

"Pingdom does not provide an API for transaction checks, so Datadog is unable to collect them."

Comment (technovangelist) [inline content/integrations/pingdom.md]: so @CharlyF is this something we show?

Comment (technovangelist) [inline content/integrations/pingdom.md]: removed the line altogether

**Pull request merged**